docs/concepts/postponed_annotations.md

```diff
@@ -1,81 +1,42 @@
-Postponed annotations (as described in [PEP563](https://www.python.org/dev/peps/pep-0563/)) "just work".
+Forward annotations (wrapped into quotes) or using the `from __future__ import annotations` [future statement]
+(as introduced in [PEP563](https://www.python.org/dev/peps/pep-0563/)) is supported:
```

Comment:
I changed _postponed annotations_ to _forward annotations_ because it feels more correct. It isn't really the annotation that is postponed, but its evaluation (as per the PEP title). I also wanted to change the actual `postponed_annotations.md` filename but this will require adding a redirect entry (which I think is fine).